src/content/capture/video-pc/js/main.js

```diff
@@ -0,0 +1,173 @@
+/*
+ *  Copyright (c) 2016 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+/* global main */
+
+'use strict';
+
+var leftVideo = document.getElementById('leftVideo');
+var rightVideo = document.getElementById('rightVideo');
+
+var stream;
+
+var pc1;
+var pc2;
+var offerOptions = {
+  offerToReceiveAudio: 1,
+  offerToReceiveVideo: 1
+};
+
+var startTime;
+
+leftVideo.onloadedmetadata = function() {
+  stream = leftVideo.captureStream();
```

Comment:
I've changed the code here: 
• Added a check for captureStream or mozCaptureStream — should probably do it the way you suggest though :).
• Removed the event listener.